layouts/css/page-modules/_footer.styl

```diff
@@ -1,7 +1,8 @@
 footer
     // display flex
     margin-top 2em
-    background-color $node-gray
+    padding 0 20px
+    background-color #404042
```

Comment:
a) Please stick to use variables for colors, this makes it easier to tune them later.
b) Footer color differs on purpose, the language selection will go there later and have the `$node-gray` background color.